src/test/java/com/jakewharton/retrofit2/adapter/kotlin/coroutines/CancelTest.kt

```diff
@@ -36,7 +36,7 @@ class CancelTest {
     val deferred = adapter.adapt(call)
     call.completeWithException(IOException())
     assertFalse(call.isCanceled)
-    assertTrue(deferred.isCompletedExceptionally)
+    assertTrue(deferred.isCancelled && deferred.isCompleted)
```

Comment:
See [`Deferred` in coroutines 0.27.0](https://github.com/Kotlin/kotlinx.coroutines/blob/0.27.0/common/kotlinx-coroutines-core-common/src/Deferred.kt#L44) for more details